docs/tutorials/viz_spiky.py

```diff
@@ -0,0 +1,118 @@
+"""
+===============
+Spiky Sphere
+===============
+In this tutorial, we show how to create a sphere with spikes.
+"""
+
+import numpy as np
+from fury import window, actor, utils, primitive
+import itertools
+
+"""
```

Comment:
Can you change the format of your documentation? It should hashtag instead of triple quotes.
You can look at the other tutorial like [this one](https://github.com/fury-gl/fury/blob/master/docs/tutorials/viz_slice.py#L16).

Same comment everywhere.